internal/follower/follower.go

```diff
@@ -118,15 +118,13 @@ func New(ref string, printer *output.Printer, conf *Config) (*Follower, error) {
 		return nil, fmt.Errorf("unable to create temporary directory: %w", err)
 	}
 
-	customPrinter := printer.WithScope(ref)
-
 	return &Follower{
 		ref:           ref,
 		tag:           tag,
 		tmpDir:        tmpDir,
 		Puller:        puller,
 		Config:        conf,
-		Printer:       customPrinter,
+		logger:        printer.Logger,
```

Comment:
`logger :  printer.Logger.With("followerName", ref)` ? So that the logger has the first argument already set, without adding it every time.
(note: i still don't know which logging library are you using though :D , i am assuming go1.21 slog)

Reply:
We can probably use same trick elsewhere.

Reply:
Can't do that. The logger is from the `pterm` package.

Reply:
Ah, sad :(